### PR TITLE
AN-147 -  Add season redesign

### PR DESCRIPTION
**Pull request by Skalakid**
Screenshot 2024-04-13 at 13 27 50

### Jira ticket

[AN-147](https://agh-student.atlassian.net/browse/AN-147)

### CODE REVIEW

**Comment by ArkadiuszMin** [comment]
Do not remove an Icon component, I'm using it in other PR

File: frontend/sumo-championship/src/components/molecules/CategoryForm/AgeCategoryFrom.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+};
+
+const AgeCategoryFrom: React.FC<props> = ({
+  onSave,
+  categories,
+  editedCategoryNumber,
+}) => {
+  const [categoryName, setCategoryName] = useState('');
+  const [errorPoints, setErrorPoints] = useState(1);
+  const [minAge, setMinAge] = useState(6);
```
Comment (ArkadiuszMin) [inline frontend/sumo-championship/src/components/molecules/CategoryForm/AgeCategoryFrom.tsx:32]: why hardcoded random default values?

File: frontend/sumo-championship/src/pages/AddSeason.tsx [hunk 2]
```diff
@@ -35,18 +35,18 @@ const AddSeason: React.FC = () => {
       name,
       startDate,
       endDate,
-      categories,
+      ageCategories: categories,
     };
 
     try {
-      await api.post('v1/season/add', newSeason)();
+      console.log(newSeason);
```
Comment (ArkadiuszMin) [inline frontend/sumo-championship/src/pages/AddSeason.tsx:42]: remove unnecessary console log

File: frontend/sumo-championship/src/components/molecules/CategoryForm/AgeCategoryFrom.tsx (hunk 1 above)
Comment (Skalakid) [reply frontend/sumo-championship/src/components/molecules/CategoryForm/AgeCategoryFrom.tsx:32]: It was like it before, thanks to it user will know min and max values

**Review by ArkadiuszMin — APPROVED** [review]

**Pull request merged**